src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3913,6 +3910,12 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="ShowPluginsOverview_NonGlobal.Content" xml:space="preserve">
     <value>Not included in global results</value>
   </data>
+  <data name="GPO_PeriodicUpdateCheckIsDisabled.Message" xml:space="preserve">
+    <value>Please check manually at regular intervals for updates.</value>
+  </data>
+  <data name="GPO_PeriodicUpdateCheckIsDisabled.Title" xml:space="preserve">
+    <value>The system administrator has disabled the automatic update checks.</value>
```

Comment:
I'd have to check - or somebody else needs to confirm - but there is more than one way to formulate this. It could also be "[this setting/auto-updates] is disabled by [system administrator/GPO]"...

Reply:
I will try to change wording.

Reply:
@Jay-o-Way 
I have checked all the different GPO info bar texts. And except from the generic version all of them starting with "The system administrator ...". So we should keep the wording here an align all the messages in a follow up PR. Maybe we can use the generic version on all places except the Run Plugin policy and the Update check policy.

![image](https://github.com/microsoft/PowerToys/assets/61519853/36bcc48e-9918-407c-9008-c4b23bd08dd1)